earth_enterprise/src/server/wsgi/serve/publish/publish_manager.py

```diff
@@ -595,12 +595,12 @@ def __GetPublishManifest(self, publish_def):
 
       # Prepare publish config.
       publish_config = libgepublishmanagerhelper.PublishConfig()
-      publish_config.fusion_host = publish_def.client_host_name
-      publish_config.db_path = publish_def.db_name
-      publish_config.stream_url = stream_url
+      publish_config.fusion_host = publish_def.client_host_name.encode()
```

Comment:
So this looks like its mostly an encode/decode issue...

Reply:
It was indeed.